book/standard_library.md

```diff
@@ -14,32 +18,218 @@ The standard library currently includes:
 To see a complete list of the commands available in the standard library, run the following:
 
 ```nu
-use std
-scope commands
-| where name =~ '^std '
-| select name usage extra_usage
-| wrap "std-lib"
+nu -c "
```

Comment:
why is this using `nu -c` instead of just calling the command?

Reply:
That's explained in the note immediately under it ;-)

> It is also run in a separate Nu subshell simply so that it is not loaded into scope in the shell you are using.

If they load it into the current shell's scope, then some of the examples (e.g., on loading with `use std/log` vs. `use std/formats *`) would be more difficult to follow.  It's not a big deal - Just keeps things a bit "cleaner" rather than messing with the scope unnecessarily.

Personally, I'd rather use a `do {}` here, but `scope` doesn't work inside a closure - It only handles the global scope.  That might be a bug (or at least an enhancement request).

Reply:
Ah, I missed that. Thanks for explaining it. Maybe I would've paid more attention to it if it were before the usage? Not sure it matters that much though.

Reply:
Ideally, this should be `do { use std; scope commands ... }`, but because of a bug in our scoping logic, `scope xxx` does not see commands inside the `do { ... }` block and the result is empty. That's something the new parser should be able to fix.